packages/wdio-utils/tests/test-framework/testInterfaceWrapper.test.js

```diff
@@ -44,7 +44,7 @@ describe('wrapTestFunction', () => {
     it('should run spec', () => {
         const specFn = jest.fn()
         const fn = wrapTestFunction(testFunction, true, 'beforeFn', () => [], 'afterFn', () => [], 'cid')
-        fn('test title', specFn)
+        fn('test title', specFn, {})
```

Comment:
Can we make this test a bit more meaningful and assert what has been passed in?
```suggestion
        fn('test title', specFn, { options: { foo: 'bar' } })
```